src/_cffi_src/openssl/hmac.py

```diff
@@ -80,4 +79,28 @@
         return 0;
 #endif
 }
+
+HMAC_CTX *Cryptography_HMAC_CTX_new(void) {
+#if OPENSSL_VERSION_NUMBER >= 0x10100000L && !defined(LIBRESSL_VERSION_NUMBER)
+    return HMAC_CTX_new();
+#else
+    /* This uses OPENSSL_zalloc in 1.1.0, which is malloc + memset */
+    HMAC_CTX *ctx = (HMAC_CTX *)OPENSSL_malloc(sizeof(HMAC_CTX));
+    memset(ctx, 0, sizeof(HMAC_CTX));
```

Comment:
HMAC_CTX_init calls EVP_MD_CTX_init on `md_ctx`, `i_ctx` and `o_ctx`. It doesn't set initial values for `md`, `key_length`, or `key`.